core/src/main/java/nl/nn/adapterframework/jms/JmsSender.java

```diff
@@ -288,7 +281,13 @@ public Message sendMessage(Message message, IPipeLineSession session, String soa
 		}
 	}
 
-	public Destination getDestination(IPipeLineSession session) throws JmsException, NamingException, JMSException {
+	public Destination getDestination(IPipeLineSession session, ParameterValueList pvl) throws JmsException, NamingException, JMSException {
```

Comment:
Mag `IPipeLineSession session` weg?

Reply:
Dan moet je in de IfsaSimulatorJmsSender misschien automatisch de getDestinationParam=replyTo zetten?